prow/plugins/assign/assign_test.go

```diff
@@ -178,8 +223,12 @@ func TestAssignComment(t *testing.T) {
 			number: 5,
 		}
 		if err := handle(fc, logrus.WithField("plugin", pluginName), ae); err != nil {
-			t.Errorf("For case %s, didn't expect error from handle: %v", tc.name, err)
-			continue
+			if tc.errExpect == nil {
+				t.Errorf("For case %s, didn't expect error from handle: %v", tc.name, err)
+				continue
+			} else if tc.errExpect.Error() != err.Error() {
```

Comment:
Never compare errors by string value. Either do a type assertion or simply check that the error exists.